test/llvm-ir-parser-tests.ts

```diff
@@ -39,7 +39,7 @@ describe('llvm-ir parseMetaNode', function () {
 
     before(() => {
         const fakeProps = new properties.CompilerProps(languages, properties.fakeProps({}));
-        compilerProps = fakeProps.get.bind(fakeProps, 'c++');
+        compilerProps = (fakeProps.get as any).bind(fakeProps, 'c++');
```

Comment:
There's probably a better way to do these

Reply:
yeah but honestly right now let's get something checked in!